ecosystem/indexer-grpc/indexer-grpc-fullnode/src/convert.rs

```diff
@@ -585,10 +585,7 @@ fn convert_signature(signature: &Signature) -> transaction::AnySignature {
             r#type: transaction::any_signature::Type::Ed25519 as i32,
             signature: s.0.clone(),
         },
-        Signature::Secp256k1Ecdsa(s) => transaction::AnySignature {
-            r#type: transaction::any_signature::Type::Secp256k1Ecdsa as i32,
-            signature: s.0.clone(),
-        },
+        Signature::Secp256k1Ecdsa(_) => panic!("Secp256k1Ecdsa conversion not supported"),
```

Comment:
Please confirm that this is okay.

Reply:
looks good. tyty!

Reply:
why do we need this? i thought this was removed

Reply:
let's remove the proto?